monitoring/uss_qualifier/scenarios/astm/netrid/common/dss/isa_validator.py

```diff
@@ -103,7 +103,7 @@ def _validate_isa(
                     )
 
         with self._scenario.check("ISA version format", dss_id) as sub_check:
-            if not all(c not in "\0\t\r\n#%/:?@[\]" for c in dss_isa.version):
+            if not all(c not in "\0\t\r\n#%/:?@[\\d]" for c in dss_isa.version):
```

Comment:
Here I believe the goal is to check for the presence of illegal characters with a simple list of illegal characters represented as a string (this is not a regex).
As such we probably want the following to check for backslashes:
```suggestion
            if not all(c not in "\0\t\r\n#%/:?@[\\]" for c in dss_isa.version):
```

(the CI fails with the version `1fqc4tmsctsdo` which contains a `d`)

Reply:
Thanks for the insight!